.travis/before_install.sh

```diff
@@ -30,16 +30,20 @@ die()
 
 case "${BUILD_TARGET}" in
 
-    linux-auto-*)
+    *-auto-*)
 	;;
 
     linux-nspr-*)
 	sudo apt-get install libnspr4 libnspr4-dev
 	;;
 
-    linux-pthreads-*)
+    *-pthreads-*)
 	;;
 
+    osx-nspr-*)
+        brew update && brew install nspr
+        ;;
+
```

Comment:
Could move this up a couple lines so it's grouped with the other nspr target.

Reply:
Thanks, addressed and pushed.